### ISSUE TITLE
Missing "Software Highlights" title to the section in Sofware Page v2

### ISSUE THREAD

**Issue by ctwhome** [research-software-directory/RSD-as-a-service#932]
During a demo, we got a quick feedback comment about the new software highlights section in v2 to include the title of "Software Highlights" on top of the Cards to distinguish between them clearly. We had it originally in the design but we didn't put it on the final version.

<img width="1500" alt="image" src="https://github.com/research-software-directory/RSD-as-a-service/assets/4195550/22d8c86f-9ffb-4809-ac5f-98b55de1a5a6">

<img width="1343" alt="image" src="https://github.com/research-software-directory/RSD-as-a-service/assets/4195550/15ba0d20-2945-47a0-aea2-f11107e15462">

**Comment by jmaassen** [comment]
Agreed, helps a lot to interpret the structure of the page as a visitor.
